xwiki-platform-core/xwiki-platform-eventstream/xwiki-platform-eventstream-default/src/main/java/org/xwiki/eventstream/internal/UntypedEventListener.java

```diff
@@ -70,15 +70,20 @@
      */
     public static final String EVENT_BINDING_NAME = "event";
 
+    /**
+     * The binding name of the event source when a validation expression is rendered.
+     */
+    public static final String SOURCE_BINDING_NAME = "source";
+
     @Inject
     private ObservationManager observationManager;
 
     @Inject
     private TemplateManager templateManager;
 
     @Inject
-    @Named("plain/1.0")
-    private BlockRenderer plainRenderer;
+    @Named("html/5.0")
```

Comment:
When using the plain renderer, the velocity code is not rendered ; I think that it’s due to the XDOM  object returned by `templateManager#execute()` (in https://github.com/xwiki/xwiki-platform/pull/587/files#diff-0a971406e804cfdf9941914cbd70ec4cL169) has the property "HTML".